src/Exception/Exception.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Opstalent\CrudBundle\Exception;
+
+/**
+ * Interface Exception
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+interface Exception
```

Comment:
is this maker interface is really needed? extending Exception class is a maker of exceptions in PHP imho

Reply:
just to catch all bundle exceptions